requirements.txt

```diff
@@ -0,0 +1,8 @@
+semantic_version
+websocket-client
```

Comment:
If we need a specific websocket-client version, we can specify it here instead of in my other PR (https://github.com/kbase/service_wizard/pull/15).